File: FrontEnd/lib/Result/PlayersResult.dart
import 'package:flutter_app/Model/Player.dart';
import 'package:flutter_app/Result/Result.dart';

class PlayersResult extends Result {
  List<Player> players;

  PlayersResult(bool success, String message) : super(success, message);

  PlayersResult.players(this.players) : super(true, "success");
}

File: noSQL_server/src/main/java/Result/PlayersResult.java
package Result;

import Model.Player;

public class PlayersResult extends Result {
  private Player[] players;

  public PlayersResult(boolean success, String message) {
    super(success, message);
  }

  public PlayersResult(Player[] players) {
    super(true, "successful");
    this.players = players;
  }

  public Player[] getData() {
    return players;
  }
}
